setup.py

```diff
@@ -53,6 +53,6 @@ def read(*rnames):
     zip_safe=False,
     install_requires=[
         'pysaml2==4.0.5',
-        'python-memcached==1.48',
+        'python-memcached>=1.48',
```

Comment:
I'd actually prefer to limit it to versions less than 2 at the same time. Also, could you please check that it really works with all newer versions released so far? Thanks.